src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -158,11 +165,12 @@ class DeepDiveLogParser extends JavaTokenParsers {
 
   def operator = "||" | "+" | "-" | "*" | "/" | "&"
   def typeOperator = "::"
-  val aggregationFunctions = Set("MAX", "SUM", "MIN", "ARRAY_ACCUM", "ARRAY_AGG")
+  val aggregationFunctions = Set("MAX", "SUM", "MIN", "ARRAY_ACCUM", "ARRAY_AGG", "COUNT")
 
-  // expressions
+  // expression
   def expr : Parser[Expr] =
-    ( lexpr ~ operator ~ expr ^^ { case (lhs ~ op ~ rhs) => BinaryOpExpr(lhs, op, rhs) }
+    ( "_" ^^ { case _ => Placeholder() }
```

Comment:
I agree it doesn't make sense to put a placeholder in the head. The idea is we use patterns for terms in the body atom, and expressions for head atom terms? 

I think we can remove placeholder in this PR, and do the pattern stuff in a separate PR, since this PR is already big.